mmm I see the doc build is still failing with:

```
  apache-airflow                                               Traceback (most recent call last):
  apache-airflow                                                 File "/usr/local/lib/python3.7/site-packages/sphinx/registry.py", line 433, in load_extension
  apache-airflow                                                   mod = import_module(extname)
  apache-airflow                                                 File "/usr/local/lib/python3.7/importlib/__init__.py", line 127, in import_module
  apache-airflow                                                   return _bootstrap._gcd_import(name, package, level)
  apache-airflow                                                 File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  apache-airflow                                                 File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  apache-airflow                                                 File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
  apache-airflow                                               ModuleNotFoundError: No module named 'sphinxcontrib.jinja'
```

and sphinx-jinja released version 1 hour ago
https://pypi.org/project/sphinx-jinja/#history

--- a/setup.py
+++ b/setup.py
@@ -617,7 +617,7 @@ devel_only = [
     'jira',
     'jsondiff',
     'mongomock',
-    'moto>=3.0.3',
+    'moto>=3.0.7',
     'parameterized',
     'paramiko',
     'pipdeptree',
@@ -640,9 +640,6 @@ devel_only = [
     'pytest-xdist',
     'python-jose',
     'pywinrm',
-    # The Responses 0.19.0 released on 07.03.2022 break our S3 tests. This limitation should be
-    # Removed when https://github.com/getsentry/responses/issues/511 is solved.
-    'responses<0.19.0',
     'qds-sdk>=1.9.6',
     'pytest-httpx',
     'requests_mock',

